### PR TITLE
Update v2.18 experimental feature statuses

### PR DESCRIPTION
**Pull request by gitbuda**
### Description

Please briefly explain the changes you made.

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

**Comment by gitbuda** [comment]
@kgolubic the statuses for each experimental feature are now up to date, feel free to merge that where/when it's ok.

File: pages/database-management/experimental-features.mdx [hunk 1]
```diff
@@ -33,19 +33,19 @@ all the necessary flags are enabled during the restart.
 
 ### High availability mode
 - **Description**: This feature enhances the resilience and fault tolerance capabilities in Memgraph Enterprise Edition, ensuring the system remains operational despite failures.
-- **Status**: Experimental. Designed to improve system reliability and uptime, but may still have stability issues.
+- **Status**: Experimental. Designed to improve system reliability and uptime, but may still have stability issues. As of now (2024-06-25), the HA feature is in the so called "development-ready" state. Feel free to test it for your production deployment, if it works, it's ok to run in production. Conditions for HA to become production-ready are a few (2-3) production deployments running 24/7 under HA setup. ETA for HA to become production-ready is end of 2024.
```
Comment (kgolubic) [inline pages/database-management/experimental-features.mdx:36]: Remove the exact date.

File: pages/database-management/experimental-features.mdx [hunk 2]
```diff
@@ -33,19 +33,19 @@ all the necessary flags are enabled during the restart.
...
+- **Status**: Experimental. Enhances data replication capabilities with added security and multi-tenancy features. It's planned to make this feature production ready for the v2.20 release.
 - **Usage**: Enable the feature using the flag `--experimental-enabled=system-replication`.
 - **Documentation**: For more details, visit the [Experimental replication documentation](/clustering/replication/experimental-replication).
 
 ### Text search
 - **Description**: This feature enables finding nodes based on property values within Memgraph, supporting both literal and regex searches and the concept of aggregation. It is not guaranteed to work correctly in use cases involving concurrent transactions and replication.
-- **Status**: Experimental. Provides text search functionalities but is not guaranteed to work correctly in use cases that involve concurrent transactions and replication.
+- **Status**: Experimental. Provides text search functionalities but is not guaranteed to work correctly in use cases that involve concurrent transactions and replication. There are no shor-term plans to improve the text-search capabilities. If you are interested in improvements, please reach out.
```
Comment (kgolubic) [inline pages/database-management/experimental-features.mdx:48]: Reach out how? Add the link to the discord.

**Review by kgolubic — APPROVED** [review]
I've updated some parts of the text. All good to go.

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**